app/javascript/packs/banner_editor.jsx

```diff
@@ -490,6 +512,35 @@ export default class BannerEditor extends React.Component {
       });
   }
 
+  incrementChannelIndex(){
+    let temp = this.state.channelIndex;
+    temp++;
+    this.setState({channelIndex: temp, loading: true},
```

Comment:
Not sure if we're linting javascript with eslint but I've seen this as lint before. Since `setState` is asynchronous we should refer to the previous state like so

```suggestion
    this.setState((prevState) => { channelIndex: (prevState.channelIndex += 1), loading: true },
```

Reply:
Done ✔️